ur_description/cfg/view_robot.rviz

```diff
@@ -100,12 +111,9 @@ Visualization Manager:
           Show Axes: false
           Show Trail: false
           Value: true
-      Name: RobotState
-      Robot Alpha: 1
-      Robot Description: robot_description
-      Robot State Topic: display_robot_state
-      Show All Links: true
-      Show Highlights: true
+      Name: RobotModel
```

Comment:
I don't know, but maybe to keep the old name of the display. Important only if you are planning to go upstream.

```suggestion
      Name: RobotState
```

Reply:
[Fix](https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/14/commits/90ae21d7a8b96abf46cd207bd73bbc7582b2eb39)